src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -84,7 +84,7 @@ namespace nupic {
                         Real minPctActiveDutyCycles=0.001,
                         UInt dutyCyclePeriod=1000,
                         Real maxBoost=10.0,
-                        Int seed=1,
+                        Int seed=-1,
```

Comment:
I believe that `-1` will result in a random seed being selected, which we do not want to be the default behavior.